src/app/components/header/header.component.scss

```diff
@@ -31,3 +31,35 @@
     }
   }
 }
+
+.dropdown {
+  position: relative;
+
+  .results {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    top: 100%;
+    left: 0;
+    right: 0;
+    z-index: 1000;
+    border: 1px solid #ccc;
+    background: white;
+    list-style: none;
+    padding: 0;
```

Comment:
I just repeat previous comment:
Maybe it will be better to make refresh all styles like `* { margin : 0, padding: 0 ....`  in main styles file for all tags

Reply:
Material UI overrides margin 0 on almost everything